### PR TITLE
roles: Fill content

### PR DESCRIPTION
**Pull request by razvand**
# Prerequisite Checklist

- [x] Read the [contribution guidelines](https://github.com/open-education-hub/ccas-internal/blob/main/CONTRIBUTING.md#pull-requests) regarding submitting new changes to the project;
- [n/a] Tested your changes against relevant architectures and platforms;
- [n/a] Updated relevant documentation (if needed).

## Description of changes

Fill contents of "Roles" chapter / section.

### CODE REVIEW

File: chapters/roles/overview/reading/README.md [hunk 1]
```diff
@@ -1,6 +1,169 @@
...
+1. Fill contents.
+   Add content, review content, improve content.
+
+The figure below presents an overview of the typical actions of content developers.
+
+![Content Developer](../media/content-developer.svg)
+
+## Educator
+
+Educators focuses exclusively on the delivery repository and is dedicated to making sure information is presented in the best way for students to learn effectively.
```
Comment (teodutu) [inline chapters/roles/overview/reading/README.md:35]:
```suggestion
Educators focus exclusively on the delivery repository and are dedicated to ensuring information is presented best for students to learn effectively.
```

File: chapters/roles/overview/reading/README.md [hunk 2]
```diff
@@ -1,6 +1,169 @@
...
+Typical educator actions are:
+
+1. Create fork of content repository as delivery repository.
+1. (Optionally) Create fork of solutions repository as solutions delivery repository.
+1. Create configuration (typically by editing the `config.yaml` file) to select and order content according to preference and specifics.
+1. Push configuration update to trigger deployment.
+   Validate deployment.
+1. Use content during educational activities with learners.
+
+Figure below presents an overview of the typical actions of educators.
```
Comment (teodutu) [inline chapters/roles/overview/reading/README.md:55]:
```suggestion
The figure below presents an overview of the typical actions of educators.
```

File: chapters/roles/overview/reading/README.md [hunk 3]
```diff
@@ -1,6 +1,169 @@
...
+
+Content is deployed automatically, triggered by updates from content developers and educators.
+Content deployment rests on default or custom configuration of [infrastructure components](../../../infrastructure/overview/reading/read.md), such as GitHub, SmileyCoin node, GitLab & [`vmchecker`](../../../infrastructure/checkers/reading/read.md) and others.
+
+Educators and content developers should not be concerned about these.
+Deployment should work out of the box.
+
+It is the responsibility of infrastructure managers to ensure proper functioning of deployment infrastructure.
+
+Figure below present an overview of the deployment steps.
```
Comment (teodutu) [inline chapters/roles/overview/reading/README.md:69]:
```suggestion
The figure below presents an overview of the deployment steps.
```

File: chapters/roles/overview/reading/README.md [hunk 4]
```diff
@@ -1,6 +1,169 @@
...
+
+![Educator](../media/deploying-content.svg)
+
+## Learner
+
+Learners are the actual beneficiaries / consumers of the educational content being deployed from the delivery repositories.
+Learners are the target group of the content being created by content developer and being customized by educators.
```
Comment (teodutu) [inline chapters/roles/overview/reading/README.md:76]:
```suggestion
Learners are the target group of the content being created by content developers and being customized by educators.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
- All images have transparent backgrounds. This sucks if they're viewed in dark mode. Add a white rectangle behind each of them in `draw.io` to force a white background.
- Remember to change all existing `README.md` files to `read.md`. Otherwise, the links to them won't work.

**Comment by razvand** [comment]
> * All images have transparent backgrounds. This sucks if they're viewed in dark mode. Add a white rectangle behind each of them in `draw.io` to force a white background.
> 
>     * Remember to change all existing `README.md` files to `read.md`. Otherwise, the links to them won't work.

@teodutu , there are no `README.md` references.

I added white background to SVG files.

**Review by teodutu — APPROVED** [review]
> @teodutu , there are no README.md references.

I was referring to files named `README.md` and referenced as `read.md`, but I see now that there aren't any thanks to #51.

**Pull request merged**